### PR TITLE
Controllers

### PR DESCRIPTION
**Pull request by waqaskanju**
In this pull
Controller for Post and user are added
- Now all users can be shown
- All posts of a user and their comments
- Detail of a single comment can be shown

### CODE REVIEW

File: app/controllers/users_controller.rb [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+  end
+
+  def detail
+    @user = User.find(params[:id])
+    @posts = @user.posts
+    @comments = @user.comments
+    @comments_count = @user.comments.count
+    @likes_count = @user.likes.count
+    @idp = params[:idp]
+  end
```
Comment (VICTOR-CHUKWU) [inline app/controllers/users_controller.rb]: - [ ] [Optiona and Important] There is no need for these methods in your user controller, what you need now is just index and show, Kindly remove

File: config/routes.rb [hunk 2]
```diff
@@ -7,4 +7,9 @@
 
   # Defines the root path route ("/")
   # root "articles#index"
+  get "users", to: "users#index"
+  get "/users/:id", to: "users#show"
+  get "/users/:id/posts", to: "users#post"
+  get "/users/:id/posts/:idp", to: "users#detail"
```
Comment (VICTOR-CHUKWU) [inline config/routes.rb]: - [ ] these routes should be for `"post#index"` and `"post#show"` kindly fix this and also fix the view folder

**Review by VICTOR-CHUKWU — CHANGES_REQUESTED** [review]
Hi @waqaskanju  👋  

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

almost

## Highlights
- Good PR
- Good commit
- All linters passed


## Required Changes ♻️
- [ ] in your view, kindly create a folder called post, for your post controller, this folder should contain `index.html.erb` and `show.html.erb` so as to match the methods in your controlle
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: app/controllers/users_controller.rb (hunk 1 above)
Comment (waqaskanju) [reply app/controllers/users_controller.rb]: removed.

File: config/routes.rb (hunk 2 above)
Comment (waqaskanju) [reply config/routes.rb]: fixed.
image

File: app/controllers/users_controller.rb (hunk 1 above)
Comment (waqaskanju) [reply app/controllers/users_controller.rb]: image

Removed from  user. added to post

**Review by ShadyShawkat — APPROVED** [review]
## Hi @waqaskanju, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To Highlight
- All linters are passing ✔️ 
- Correct use of GitFlow ✔️ 
- Descriptive commit messages ✔️ 
- Professional looking pull request ✔️ 
- Your readme looks professional ✔️ 

Image

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**